### PR TITLE
[TF 2.0 docs] Add description for activation functions

### PR DESCRIPTION
**Pull request by aweers**
Add descriptions to softplus, softsign and tanh. I am not a native English speaker, so maybe there are some spelling errors.

### CODE REVIEW

File: tensorflow/python/keras/activations.py [hunk 1]
```diff
@@ -122,7 +121,11 @@ def selu(x):
 
 @keras_export('keras.activations.softplus')
 def softplus(x):
-  """Softplus activation function.
+  """The Softplus activation function is a smoothed version of
+  the relu function. For negative values the function returns a 
+  value close to zero, for positve values the result of
```
Comment (kyscg) [inline tensorflow/python/keras/activations.py]:
```suggestion
  value close to zero, for positive values the result of 
```

File: tensorflow/python/keras/activations.py [hunk 2]
```diff
@@ -122,7 +121,11 @@ def selu(x):
 
 @keras_export('keras.activations.softplus')
 def softplus(x):
-  """Softplus activation function.
+  """The Softplus activation function is a smoothed version of
+  the relu function. For negative values the function returns a 
+  value close to zero, for positve values the result of 
+  the function is similiar to the result a linear function
```
Comment (kyscg) [inline tensorflow/python/keras/activations.py]:
```suggestion
  the function is similar to what a linear function 
```

File: tensorflow/python/keras/activations.py [hunk 3]
```diff
@@ -135,7 +138,12 @@ def softplus(x):
 
 @keras_export('keras.activations.softsign')
 def softsign(x):
-  """Softsign activation function.
+  """The Softsign activation function is similiar to the
```
Comment (kyscg) [inline tensorflow/python/keras/activations.py]:
```suggestion
  """The Softsign activation function is similar to the 
```

File: tensorflow/python/keras/activations.py [hunk 4]
```diff
@@ -135,7 +138,12 @@ def softplus(x):
 
 @keras_export('keras.activations.softsign')
 def softsign(x):
-  """Softsign activation function.
+  """The Softsign activation function is similiar to the 
+  tanh function. For large values the function returns 
+  values close to 1 and for small values the function 
+  returns values close to -1. Like tanh the Softsign 
+  activation function goes through the origin, but 
+  has a smaller slope than tanh.
```
Comment (kyscg) [inline tensorflow/python/keras/activations.py]:
```suggestion
  has a smaller slope. 
```

File: tensorflow/python/keras/activations.py [hunk 5]
```diff
@@ -171,7 +179,10 @@ def relu(x, alpha=0., max_value=None, threshold=0):
 
 @keras_export('keras.activations.tanh')
 def tanh(x):
-  """Hyperbolic Tangent activation function.
+  """The Hyperbolic Tangent activation function is a similiar to the
```
Comment (kyscg) [inline tensorflow/python/keras/activations.py]:
```suggestion
  """The Hyperbolic Tangent activation function is a similar to the 
```

File: tensorflow/python/keras/activations.py [hunk 6]
```diff
@@ -171,7 +179,10 @@ def relu(x, alpha=0., max_value=None, threshold=0):
 
 @keras_export('keras.activations.tanh')
 def tanh(x):
-  """Hyperbolic Tangent activation function.
+  """The Hyperbolic Tangent activation function is a similiar to the 
+  sigmoid activation function. It has also a smooth curve from its
```
Comment (kyscg) [inline tensorflow/python/keras/activations.py]:
```suggestion
  sigmoid activation function. It also is a smooth curve ranging within 
```

File: tensorflow/python/keras/activations.py [hunk 7]
```diff
@@ -171,7 +179,10 @@ def relu(x, alpha=0., max_value=None, threshold=0):
 
 @keras_export('keras.activations.tanh')
 def tanh(x):
-  """Hyperbolic Tangent activation function.
+  """The Hyperbolic Tangent activation function is a similiar to the 
+  sigmoid activation function. It has also a smooth curve from its 
+  minimum to its maximum value. The difference to the sigmoid function
```
Comment (kyscg) [inline tensorflow/python/keras/activations.py]:
```suggestion
  its minimum and maximum value. The difference from the sigmoid function 
```

**Review by kyscg — CHANGES_REQUESTED** [review]
Nice contribution, could you please make the following changes?

I've made them out for you, just accept the suggested changes.

Cheers!!

**Comment by kyscg** [comment]
We've signed the CLA's. @rthadur, could you set CLA's to yes and review please

Thanks..

**Comment by aweers** [comment]
Thanks, @kyscg for the improvements! ;)

**Review by kyscg — DISMISSED** [review]

**Comment by kyscg** [comment]
@rthadur, gentle ping to review and set CLA's to yes

**Comment by rthadur** [comment]
@aweers thank you for your contribution , please make sure all the contributors have signed CLA

**Comment by aweers** [comment]
@rthadur @kyscg and I signed the cla and we are the only contributors ;)

**Comment by kyscg** [comment]
@rthadur @akshaym Gentle ping to review

**Review by fchollet — CHANGES_REQUESTED** [review]
Thank you for the PR. You should format docstrings according to the TensorFlow style conventions; in particular, the docstring should start with a single line ending in a period, followed by 2 line breaks.

**Comment by aweers** [comment]
Hi @fchollet, thanks for your suggestions! I hope now everything matches the style conventions. ;)

**Comment by mellanox-github** [comment]
Can one of the admins verify this patch?

**Comment by rthadur** [comment]
@aweers can you please resolve conflicts ?

**Comment by tensorflowbutler** [comment]
It has been 15 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by tensorflowbutler** [comment]
It has been 43 days that this pull-request has stalled. Please create a new pull-request with the requested changes.
